bpf/lib/common.h

```diff
@@ -445,6 +446,16 @@ enum {
  */
 #define MARK_MAGIC_SNAT_DONE		0x1500
 
+/* MARK_MAGIC_IPIP_DONE can overlap with MARK_MAGIC_SNAT_DONE with both being
+ * mutual exclusive.
+ */
+#define MARK_MAGIC_IPIP_DONE		MARK_MAGIC_SNAT_DONE
+
+/* MARK_MAGIC_HEALTH can overlap with MARK_MAGIC_DECRYPT with both being
+ * mutual exclusive.
```

Comment:
Not going to NAK based on this but IMO we need to unwind this decrypt mutual exclusive problem at some point.